src/components/app-root/MapsTabBody/MapsTabBody.js

```diff
@@ -79,26 +74,23 @@
 // again.
 //
 // There are several possible approaches to this problem, but the one that so
-// far has yielded the best results is to monitor the size of an (really any)
-// element in the Maps tab and redraw the maps when the size changes. This may
-// cause some unnecessary redraws when the window size is changed, but that
-// is irrelevant, and may not even be causing Leaflet to do any actual work.
-//
-// To monitor element size, we use React Measure, which has a very convenient
-// `onResize()` callback prop.
```

Comment:
We're no longer using `Measure` because setting the `Tab` options `mountOnEnter` and `unmountOnExit`?